src/shortcuts.js

```diff
@@ -147,9 +147,11 @@ export const handleKeyboard = e => {
   const state = store.getState()
   if (state.showModal === 'welcome' || state.showModal === 'help' || state.showModal === 'feedback') return
 
+  const keyCode = e.which || e.keyCode || 0
+  const keyPressed = (e.shiftKey && keyCode > 64 && keyCode < 91) ? e.key.toLowerCase() : e.key
```

Comment:
I think the conditional is unnecessary. Wouldn't `e.key.toLowerCase()` always work?

Reply:
> I think the conditional is unnecessary. Wouldn't `e.key.toLowerCase()` always work?

Yes, I found a case like key to delete a thought is 'Backspace' which becomes 'backspace'. So I put the above condition.

Reply:
In that case, I recommend lowercasing the first letter.

Reply:
Sure. Will lowercase all shortcut keys.

Reply:
No, I mean replace line 151 with:

```
const keyPressed = e.key[0].toLowerCase() + e.key.slice(1)
```

Reply:
I am not getting you here, help me understand how this going to work. Like implementing 
`const keyPressed = e.key[0].toLowerCase() + e.key.slice(1)`
will lowercase first letter i.e 'Tab' key will become 'tab' and we are comparing it with our defined shortcut which is key='Tab'.

Reply:
Thank you, good question. We should `.toLowerCase` the shortcut `key` in the comparison. This will be more resilient for the addition new shortcuts.

Reply:
Thank You. Will do the same.